data/language/dutch.txt

```diff
@@ -835,7 +835,7 @@ STR_0833    :{SMALLFONT}{BLACK}Spel pauzeren
 STR_0834    :{SMALLFONT}{BLACK}Schijf- en spelopties
 STR_0835    :Kan het spel niet initialiseren
 STR_0836    :Kan het spel niet in geminimaliseerde toestand starten
-STR_0837    :Kan het grafisch systeem niet starten
+STR_0837    :Kan het grafische systeem niet starten
```

Comment:
Hoewel je verandering niet incorrect is, verdient 'grafisch systeem' de voorkeur, omdat het een vaste verbinding is (d.w.z. dat het zich in de praktijk als één zelfstandig naamword gedraagt).